src/NosCore.Parser/Parsers/NpcMonsterParser.cs

```diff
@@ -0,0 +1,495 @@
+                {nameof(NpcMonsterDto.MaxMp), chunk => Convert.ToInt32(chunk["HP/MP"][0][3]) + Convert.ToByte(chunk["RACE"][0][2]) == 0 ? _basicPrimaryMp[Level(chunk)] : _basicSecondaryMp[Level(chunk)]},
+                {nameof(NpcMonsterDto.Xp), chunk =>  Math.Abs(Convert.ToInt32(chunk["EXP"][0][2]) + _basicXp[Level(chunk)])},
+                {nameof(NpcMonsterDto.JobXp), chunk => Convert.ToInt32(chunk["EXP"][0][3]) + _basicJXp[Level(chunk)]},
+                {nameof(NpcMonsterDto.IsHostile), chunk => chunk["PREATT"][0][2] != "0" },
+                {nameof(NpcMonsterDto.NoticeRange), chunk => Convert.ToByte(chunk["PREATT"][0][4])},
+                {nameof(NpcMonsterDto.Speed), chunk => Convert.ToByte(chunk["PREATT"][0][5])},
+                {nameof(NpcMonsterDto.RespawnTime), chunk => Convert.ToInt32(chunk["PREATT"][0][6])},
+                {nameof(NpcMonsterDto.CloseDefence), chunk =>  Convert.ToInt16((Convert.ToInt16(chunk["PREATT"][0][2]) - 1) * 2 + 18)},
+                {nameof(NpcMonsterDto.DistanceDefence), chunk =>  Convert.ToInt16((Convert.ToInt16(chunk["PREATT"][0][2])- 1) * 3 + 17)},
+                {nameof(NpcMonsterDto.MagicDefence), chunk => Convert.ToInt16((Convert.ToInt16(chunk["PREATT"][0][2]) - 1) * 2 + 13)},
+                {nameof(NpcMonsterDto.DefenceDodge), chunk =>  Convert.ToInt16((Convert.ToInt16(chunk["PREATT"][0][2])- 1) * 5 + 31)},
+                {nameof(NpcMonsterDto.DistanceDefenceDodge), chunk => Convert.ToInt16((Convert.ToInt16(chunk["PREATT"][0][2]) - 1) * 5 + 31)},
+                {nameof(NpcMonsterDto.VNumRequired), chunk => chunk["SETTING"][0][4] == "0" ? 0 : Convert.ToInt32(chunk["SETTING"][0][4])},
+                {nameof(NpcMonsterDto.AmountRequired), chunk => chunk["SETTING"][0][4] == "0" ? 0 : 1},
+                {nameof(NpcMonsterDto.AttackClass), chunk => Convert.ToByte(chunk["ZSKILL"][0][2])},
+                {nameof(NpcMonsterDto.BasicRange), chunk => Convert.ToByte(chunk["ZSKILL"][0][3])},
+                {nameof(NpcMonsterDto.BasicArea), chunk => Convert.ToByte(chunk["ZSKILL"][0][5])},
+                {nameof(NpcMonsterDto.BasicCooldown), chunk => Convert.ToInt16(chunk["ZSKILL"][0][6])},
+                {nameof(NpcMonsterDto.AttackUpgrade), chunk => Convert.ToByte(LoadUnknownData(chunk) == 1?chunk["WINFO"][0][2]:chunk["WINFO"][0][4])},
+                {nameof(NpcMonsterDto.DefenceUpgrade), chunk => Convert.ToByte(LoadUnknownData(chunk) == 1? chunk["WINFO"][0][2]:chunk["AINFO"][0][3])},
+                {nameof(NpcMonsterDto.BasicSkill), chunk => Convert.ToInt16(chunk["EFF"][0][2])},
+                {nameof(NpcMonsterDto.VNumRequired), chunk => Convert.ToInt16(ShouldLoadPetinfo(chunk) ? Convert.ToInt16(chunk["PETINFO"][0][2]) : 0)},
+                {nameof(NpcMonsterDto.AmountRequired), chunk => Convert.ToByte((ShouldLoadPetinfo(chunk) ? Convert.ToInt16(chunk["PETINFO"][0][2]) : 0) != 0 ? Convert.ToByte(chunk["PETINFO"][0][3]) : 0)},
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Split this 204 characters long line (which is greater than 200 authorized).](https://app.codacy.com/app/NosCoreIO/NosCore/pullRequest?prid=4512441)